dataplane/worker_gc.cpp

```diff
@@ -931,6 +938,40 @@ inline bool worker_gc_t::is_timeout(const uint16_t timestamp,
 	return ((uint16_t)(current_time - timestamp) > timeout);
 }
 
+inline uint32_t worker_gc_t::get_acl_tcp_timeout(const uint8_t flags, const dataplane::globalBase::fw_state_config_t& fw_state_config)
+{
+	if ((flags & RTE_TCP_ACK_FLAG) && (flags & RTE_TCP_SYN_FLAG))
+	{
+		return fw_state_config.tcp_ack_timeout;
```

Comment:
Following the condition above this should be named as `tcp_syn_ack_timeout`, isn't it?